db/migrate/20170812152912_add_unique_constraint_to_craftsmen_employment_id.rb

```diff
@@ -0,0 +1,18 @@
+class AddUniqueConstraintToCraftsmenEmploymentId < ActiveRecord::Migration
+  def change
```

Comment:
I didn't know there was a syntax to do it this way. Typically I'd write two methods `up` and `down` and rails knows to use `up` during migration and `down` during rollback.